ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -28,21 +28,25 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex)
 
             var pageSize = 50;
 
-            PageIndex = pageIndex ?? 1;
+            IQueryable<Notification> draftsIQ = service.GetBaseNotificationIQueryableByNotificationStatus(new List<NotificationStatus>() {NotificationStatus.Draft});

```

Comment:
I like the generic appraoch with the list, but the line length is suffering here - can we cut it up a little to make it more readable? :)
Maybe shortenning that method name a bit too, as its getting unwieldy (how about `GetBaseQueryableNotificationByStatus`?